backend/app/tests/crud/test_crud_content.py

```
@@ -0,0 +1,149 @@
+import uuid
+from datetime import datetime, timezone
+from typing import List, Optional
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)